properties/pkg/properties_test.go

```diff
@@ -593,13 +562,13 @@ func TestProperties_Equal(t *testing.T) {
 			nextKeyIndex: 3,
 		}
 
-		if p.Equal(&expected) {
-			t.Errorf("Comparing Properties two objects with different set of keys must not be equal!")
-		}
+		g.Expect(p.Equal(expected)).ShouldNot(BeTrue())
```

Comment:
```suggestion
		g.Expect(p).ShouldNot(BeEquivalentTo(expected))
```

Reply:
I'd not change this as here we need to test the correctness of `Equal` method which does not take the `comment` field of `Property` struct into the consideration when comparing two `Properties` objects.